nodejs/week1/avg.js

```diff
@@ -0,0 +1,32 @@
+// Here we first get command line arguments, ignoring the first two elements
+const args = process.argv.slice(2);
+
+// But if there're basically no argument, then we print a message and exit
+if (args.length === 0) {
+  console.log("Please provide numbers");
+  process.exit(1);
+}
+
+// Here with this loop we convert each comment line argument from strings to numbers
+const numbers = [];
```

Comment:
suggestion: instead of using Number(args[i]) in a loop, consider using args.map(Number) to convert all arguments to numbers in a single step - a more consise approach.